.github/workflows/apps/appsec-test-contrib-submodules.sh

```diff
@@ -41,7 +41,16 @@ fi
 
 $runner "$JUNIT_REPORT.xml" "." ./appsec/... ./internal/appsec/...
 
-SCOPES=("gin-gonic/gin" "google.golang.org/grpc" "net/http" "gorilla/mux" "go-chi/chi" "go-chi/chi.v5" "labstack/echo.v4")
+SCOPES=(
+  "gin-gonic/gin" \
+  "google.golang.org/grpc" \
+  "net/http" "gorilla/mux" \
+  "go-chi/chi" "go-chi/chi.v5" \
+  "labstack/echo.v4" \
+  "99designs/gqlgen" \
+  "graphql-go/graphql" \
+  "graph-gophers/graphql-go"
```

Comment:
This adds new contribs as well. Is this intended?
Asking since it's not reflected in the PR description.

Reply:
Yes, they were missing in our list of contribs to test.
Adding it in the PR description